src/foremast/utils/templates.py

```diff
@@ -46,13 +47,14 @@ def get_template_object(template_file=''):
     jinja_lst = []
 
     if TEMPLATES_PATH:
-        external_templates = os.path.expanduser(TEMPLATES_PATH)
+        external_templates = pathlib.Path(TEMPLATES_PATH).expanduser().resolve()
         assert os.path.isdir(external_templates), 'External template path "{0}" not found'.format(external_templates)
         jinja_lst.append(external_templates)
 
     jinja_lst.append(LOCAL_TEMPLATES)
+    jinja_templates = [str(path) for path in jinja_lst]
```

Comment:
Technically this is a list of Jinja template paths.

Reply:
:+1: Updated.